variables.tf

```
@@ -55,6 +55,12 @@ variable "bucket_force_destroy" {
   default = false
 }
 
+variable "bucket_region" {
+  type        = string
+  default     = "us"
```

shouldn't these values be capitalized? 🤔 https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#location
```suggestion
  default     = "US"
```

---

@afiune my latest commit now reflects the changes